smartsim/log.py

```diff
@@ -214,7 +213,7 @@ def get_logger(
     if log_level:
         logger.setLevel(log_level)
     else:
-        log_level = user_log_level
+        log_level = _translate_log_level(user_log_level)
     coloredlogs.install(level=log_level, logger=logger, fmt=fmt, stream=sys.stdout)
```

Comment:
Oh wow! Yeah, now that you pointed out this "psuedo-caching" behavior of loggers and log levels, this definitely feels wrong. At the least, I really don't like how we call `logger.setLevel` in only one of these branches.

Definitely might be worth a more in-depth refactor later!